client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -619,8 +519,10 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                 Object.values((user.userType === userType.ADMIN || user.userType === userType.SUPER_ADMIN) ? adminCols : userCols).map((key: string) => (
                                                     <TableCell
                                                         classes={{ root: classes.tableCellRoot }}
-                                                        padding='none'
                                                         className={getRegularCellStyle(index, key , isGroupShown).join(' ')}
+                                                        onMouseOver={() => key === TableHeadersNames.investigatorName && setEpiNumOnInvestigatorNameHover(indexedRow.epidemiologyNumber)}
```

Comment:
You should move this onMouseOver events to investigator name cell instead of each cell in the table.

Reply:
fixed